docs/hello_nextflow/02_hello_world.md

```diff
@@ -270,52 +270,36 @@ Learn how to pass parameters to the workflow from the command line.
 
 ---
 
-## 4. Use a command line parameter for naming the output file
+## 4. Write the result out to a file
```

Comment:
Additionally I think this new point should be folded into the previous section because it's the immediate continuity. So would make it "#### 4." and remove the previous takeaway/next steps block (making sure to combine the contents if needed)